src/Uno.UI.Wasm/ts/Windows/UI/ViewManagement/ApplicationView.ts

```diff
@@ -0,0 +1,20 @@
+﻿namespace Windows.UI.ViewManagement {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['namespace' and 'module' are disallowed](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4878148)

Reply:
I'm not sure what to do with this one 🤔 , `namespace` are used almost in all `.ts` files - like https://github.com/unoplatform/uno/blob/master/src/Uno.UI.Wasm/ts/Windows/Devices/Geolocation/Geolocator.ts or https://github.com/unoplatform/uno/blob/master/src/Uno.UI.Wasm/ts/Windows/Storage/StorageFolder.ts